### ISSUE TITLE
[nsfw] nicesss.com: popups

### ISSUE THREAD

**Issue by Forberg21** [uBlockOrigin/uAssets#17101]
### Prerequisites

- [X] I read and understand the [policy about what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets).
- [X] I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)
- [X] I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png), or I have verified that the issue was not caused by removing any of the default lists.
- [X] I did not enable external filter lists, or I have verified that the issue still occurs without enabling external filter lists.
- [X] I do not have custom filters/rules, or I have verified that the issue still occurs without custom filters/rules.
- [X] I am not using uBlock Origin (uBO) along with other [similarly-purposed blockers](https://twitter.com/gorhill/status/1033706103782170625).
- [X] I have verified that the web browser's built-in blocker or [DNS blocking](https://en.wikipedia.org/wiki/DNS_blocking) (standalone or through a VPN) is not causing the issue.
- [ ] I did not answer truthfully to **ALL** the above checkpoints.

### URL address of the web page

`https://www.nicesss.com/archives/xr/xiurw/124286.html`

### Category

popups

### Description

Sorry for keep asking for help

Don't know how they manage to break ublock, but after your feedbacks, only few days work but now it doesn't work

keep showing fucking ads with new popups.

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.47.4
Chromium: 110
filterset (summary): 
  network: 86558
  cosmetic: 42216
  scriptlet: 17460
  html: 0
listset (total-discarded, last-updated): 
  default: 
    user-filters: 0-0, never
    KOR-1: 4244-95, never
    easylist: 63372-534, never
    easyprivacy: 31007-1180, never
    plowe-0: 3751-590, never
    ublock-abuse: 78-0, never
    ublock-badware: 4611-0, never
    ublock-filters: 32807-86, never
    ublock-privacy: 287-0, never
    ublock-quick-fixes: 340-16, never
    ublock-unbreak: 1922-0, never
    urlhaus-1: 6492-0, never
filterset (user): [empty]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  allReadyAfter: 1213 ms
  maxAssetCacheWait: 8 ms
popupPanel: 
  blocked: 0
```
</details>

**Comment by mapx-** [comment]
1. don't keep opening new issues => use the old one, we see the comments
2. your installation is broken, follow ALL indications here:

- clear cookies / localStorage/ cache for the tested page
- reset uBO, don't modify any uBO setting, update all the filter lists, don't add any other list (other than your regional list)
- disable any custom filter for that site
- disable any other extension / addon in your browser, disable builtin blockers /shields/ tracking protection in brave/ opera/ vivaldi/ edge/ firefox etc
- if Opera browser: open opera://extensions/ and check the "allow access to search page results" option
- disable (if any) the web shields in your antivirus / antimalware software / other blocking software
- are you using a DNS which features adblocking (like AdGuard DNS)?=> test using another DNS (google for example =>  IP addresses 8.8.8.8 & 8.8.4.4)
- restart the system, reopen the browser, test again.

**Comment by Forberg21** [comment]
Sorry for new issue

Using dns might be the problem. changing helped THX

**Comment by Forberg21** [comment]
clear cookies / localStorage/ cache for the tested page => Done
reset uBO, don't modify any uBO setting, update all the filter lists, don't add any other list (other than your regional list)
disable any custom filter for that site => Reinstall Done
disable any other extension / addon in your browser, disable builtin blockers /shields/ tracking protection in brave/ opera/ vivaldi/ edge/ firefox etc => using chrome. No other extensions except "Editthiscookie"(which never touched), "HTTPSeverywhere"(disabled for the site)
if Opera browser: open opera://extensions/ and check the "allow access to search page results" option
disable (if any) the web shields in your antivirus / antimalware software / other blocking software =>Only Window10 original antivirus 
are you using a DNS which features adblocking (like AdGuard DNS)?=> test using another DNS (google for example => IP addresses 8.8.8.8 & 8.8.4.4) =>Nope. 
restart the system, reopen the browser, test again. =>Done.

Still happening. Don't know why

**Comment by mapx-** [comment]
Not only reinstalling but also => update all the filter lists

**Comment by Forberg21** [comment]
> Not only reinstalling but also => update all the filter lists

Thx. Done. Seems working. If this happens again, I will follow those instruction again

After done it, if happens, I will comment on you again in this issue. Thx again

**Comment by garry-ut99** [comment]
>  * [x]  I forced an update of my filter lists. (Click the ["Purge all caches"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches) button while holding the 'Shift' key, then click the ["Update now"](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now) button.)

No, you didn't, in [all three of your reports](https://github.com/uBlockOrigin/uAssets/issues?q=is%3Aissue+nicesss.com)

> update all the filter lists Done

No, you still didn't.

> Still happening. Don't know why

Because you didn't do what you claimed you did.

> Done. Seems working.

Finally, congratulations.

> If this happens again, I will follow those instruction again

If you had followed from the beginning, then you wouldn't even need to file an issue.

<hr>

> https://github.com/uBlockOrigin/uAssets/issues/16994 + https://github.com/uBlockOrigin/uAssets/issues/17101 :
>  * [x]  I verified that this issue is not a duplicate. (Use this [button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out.)

No, you didn't, you created 2 duplicates of your previous reports.
<hr>

> (Quote): To avoid burdening **VOLUNTEERS** with invalid or duplicate reports, disregarding these steps **WILL** get your account blocked on this repository temporarily, then permanently, with repeated disregard for the template.
